zerver/lib/digest.py

```diff
@@ -276,7 +277,9 @@ def get_slim_stream_map(stream_ids: Set[int]) -> Dict[int, Stream]:
     return {stream.id: stream for stream in streams}
 
 
-def bulk_get_digest_context(users: List[UserProfile], cutoff: float) -> Dict[int, Dict[str, Any]]:
+def bulk_get_digest_context(
+    users: Union[Sequence[UserProfile], QuerySet[UserProfile]], cutoff: float
```

Comment:
Could be `users: Collection[UserProfile]` if we changed `realm = users[0].realm` to `realm = next(iter(users)).realm`.